queries/dart/highlights.scm

```diff
@@ -105,13 +105,11 @@
 (final_builtin) @constant.builtin
 
 ((identifier) @type
- (#match? @type "^_?[A-Z]"))
+ (#match? @type "^_?[A-Z][a-z]"))
```

Comment:
I used `[A-Z].*[a-z]` in one of the other languages to also catch stuff like `IBlock` and `VBuffer`

Reply:
Thanks for the pointer just tweaked it